packages/suite/package.json

```diff
@@ -28,9 +28,9 @@
         "@suite-common/connect-init": "*",
         "@suite-common/fiat-services": "*",
         "@suite-common/formatters": "*",
+        "@suite-common/in-app-notifications": "*",
```

Comment:
Not sure about the name of the package

Reply:
Other options in my mind `inapp-notifications`, `toast-notifications`

Reply:
`@suite-common/toasts` or `@suite-common/toast-notifications` sounds okay

Reply:
renamed to @suite-common/toast-notifications